modules/mobile/app/models/mobile/v0/adapters/claims_overview.rb

```diff
@@ -65,7 +66,8 @@ def parse_appeal(entry)
               completed: !entry['attributes']['active'],
               date_filed: entry['attributes']['events'][filed_index]['date'],
               updated_at: entry['attributes']['events'].last['date'],
-              display_title: get_appeals_display_title(subtype, entry['attributes']['programArea'])
+              display_title: get_appeals_display_title(subtype, entry['attributes']['programArea']),
+              decision_letter_sent: false
```

Comment:
appeal decision letters aren't included in decision letter listing right now. Will be added later.